paddle/operators/detail/CMakeLists.txt

```diff
@@ -1 +1,5 @@
 grpc_library(sendrecvop_grpc SRCS recv_impl.cc send_impl.cc PROTO send_recv.proto DEPS lod_tensor selected_rows)
+set_source_files_properties(
```

Comment:
Put these lines in the function `grpc_library`? This is only needed for the grpc dependencies.

Reply:
Delete these lines and add flag `-Wno-non-virtual-dtor ` in send/recv prpoerties.